samples/cellular/modem_shell/src/ping/icmp_ping.c

```diff
@@ -52,6 +54,15 @@ enum ping_rai {
 	PING_RAI_LAST_PACKET
 };
 
+int64_t reg_time; /* When LTE reg status was updated last time:
+		   * used to detect the cases when we need to re-read
+		   * PDN context info to update ping ip addresses in case of possible
+		   * changes.
+		   */
```

Comment:
I would move the comment above the variable

Reply:
Done